website/src/releases/hostname-migration.mdx

```diff
@@ -0,0 +1,70 @@
+---
+date: 2020-06-10
+title: Migrate Custom Applications to the new host naming scheme
+description: |
+  The Merchant Center is adopting a new host naming scheme and deprecating the old hostnames.\
+  Therefore, Custom Applications must be updated to support both hostnames, for backwards compatibility.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Recently we [introduced a new host naming scheme](/main-concepts/api-gateway#cloud-regions) for the commercetools APIs, including the Merchant Center, to increase region and cloud provider transparency.
+
+While the previous hostnames such as `*.sphere.io` (Google Cloud, Belgium), `*.commercetools.com` (Google Cloud, Belgium), and `*.commercetools.co` (Google Cloud, Iowa) continue to be operational, we **strongly recommend** to migrate to the new hostname structure.
+
+In regards to Custom Applications, the [runtime configuration](/deployment/runtime-configuration) needs to be updated to **support both the new and the legacy hostnames**, to allow users to log in with the new Merchant Center hostname and to be backwards compatible with the legacy hostnames.
+
```

Comment:
1.) The comma after` legacy hostnames` should be eliminated.  

2.) A comma _may_  be added after the `new Merchant Center hostname...`, but it comes down to preference. 
                 The "and" naturally lets the audience take a figurative breath in the sentence, but a comma wouldn't 
                  hurt nor be grammatically incorrect.

(thanks to Charles Thompson for looking at this)